src/api/app/policies/bs_request_policy.rb

```diff
@@ -7,4 +7,8 @@ def create?
 
     false
   end
+
+  def can_handle_request?(is_target_maintainer, is_author)
+    return true if record.state.in?([:new, :review, :declined]) && (is_target_maintainer || is_author)
```

Comment:
This code can be simplified:

```suggestion
    record.state.in?([:new, :review, :declined]) && (is_target_maintainer || is_author)
```

Reply:
copy/paste is a bad attitude I need to stop at some point :sweat_smile: